app/controllers/venues_controller.rb

```diff
@@ -0,0 +1,7 @@
+class VenuesController < ApplicationController
+
+  def show
+    @venue = Venue.find_by(id: params[:id])
+    @events = Event.active_events(@venue)
```

Comment:
But awesome logic of knowing we only want the active events for that particular venue! :sparkles: 

Also, I'm not sure if we would have time but it might be cool to make a slug of the venue name instead of using the id so that way we could type in a path to a venue. This works though!